src/test/java/com/synopsys/integration/alert/util/AlertIntegrationTest.java

```diff
@@ -35,7 +34,6 @@
 @ContextConfiguration(classes = { Application.class, ApplicationConfiguration.class, DatabaseDataSource.class, DescriptorMocker.class })
 @TestPropertySource(locations = "classpath:spring-test.properties")
 @WebAppConfiguration
-@Transactional
```

Comment:
This being here had gotten to the point that tests were becoming impossible to debug. There were ~40 integration tests and about half of them already had `@Transactional` at the class level anyway.